docs/ja/docs/advanced/additional-status-codes.md

```diff
@@ -0,0 +1,38 @@
+# 追加のステータスコード
+
+デフォルでは、 **FastAPI** は `JSONResponse` を使ってレスポンスを返します。その `JSONResponse` の中には、 *path operation* が返した内容が入ります。
+
+それは、デフォルトのステータスコードか、 *path operation* でセットしたものを利用します。
+
+## 追加のステータスコード
+
+メインのステータスコードとは別に追加のステータスコードを返したい場合は、追加のステータスコードが設定された `JSONResponse` のような `Response` を直接返します。
+
+例えば、itemを更新し、成功した場合は200 "OK"のHTTPステータスコードを返す *path operation* を作りたいとします。
+
+しかし、新しいitemも許可したいとします。itemが存在しない場合は、それらを作成して201 "Created"を返すとします。
```

Comment:
```suggestion
しかし、新しいitemも許可したいです。itemが存在しない場合は、それらを作成して201 "Created"を返します。
```

You can see from the previous sentence that it is a hypothetical story. So I think this is more natural.